dist/s_comment.py

```diff
@@ -0,0 +1,67 @@
+import re, sys
```

Comment:
@agorrod @keithbostic 
This is the requested script to fill comments to the new line length.

It works well for the most part but I doubt we'll ever get it to the point where it'll be fine to run this unsupervised. For example:
* We don't always paragraph comments in the same way.
* Some comments contain numbered lists/bullet points.
* Comments within macros.

If you both agree, I'd rather just use this script as a one time thing to do most of the legwork rather than making it first class and keeping it under `dist/`.